src/wagtail_ai/views.py

```diff
@@ -79,8 +80,8 @@ def process(request):
     if not text:
         return JsonResponse(
             {
-                "error": "No text provided - please enter some text before using AI \
-                    features"
+                "error": "No text provided - "
+                "please enter some text before using AI features"
```

Comment:
**Nitpick:** This multi-lie string formatting in a dict seems a bit annoying. What do you think? I wonder if we could set the string in a separate variable or use parentheses.

Reply:
done 45b75e8